onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -19,6 +19,8 @@
 namespace onnxruntime {
 namespace qnn {
 
+static std::mutex qnn_log_mutex_;
```

Comment:
better to use function static rather than static global.
and use OrtMutex
e.g. https://github.com/microsoft/onnxruntime/blob/4324d2173b44eda7967ee2348af7c9134f8c1350/onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc#L361

btw, is this lock required by qnn logging api?

Reply:
removed

---
In reply to: [1199310134](https://github.com/microsoft/onnxruntime/pull/16019#discussion_r1199310134) [](ancestors = 1199310134)